### PR TITLE
Issue 7044: Do not import cookies from Chrome and Firefox.

### PR DESCRIPTION
**Pull request by iefremov**
Note: we do not import cookies from Safari, so nothing to disable.

Fix https://github.com/brave/brave-browser/issues/7044
## Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/brave-browser/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [x] Added/updated tests for this change (for new code or code which already has tests).
- Verified that these changes build without errors on
  - [ ] Android
  - [ ] iOS
  - [x] Linux
  - [ ] macOS
  - [ ] Windows
- Verified that these changes pass automated tests (unit, browser, security tests) on
  - [ ] iOS
  - [x] Linux
  - [ ] macOS
  - [ ] Windows
- [ ] Verified that all lint errors/warnings are resolved (`npm run lint`)
- [x] Ran `git rebase master` (if needed).
- [x] Ran `git rebase -i` to squash commits (if needed).
- [x] Tagged reviewers and labelled the pull request as needed.
- [ ] Request a security/privacy review as needed.
- [ ] Add appropriate QA labels (QA/Yes or QA/No) to include the closed issue in milestone
- [ ] Public documentation has been updated as necessary. For instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protection-Mode
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-compatibility-issues-with-tracking-protection
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide

## Reviewer Checklist:

- [ ] New files have MPL-2.0 license header.
- [ ] Request a security/privacy review as needed.
- [ ] Adequate test coverage exists to prevent regressions 
- [ ] Verify test plan is specified in PR before merging to source

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on.
- [ ] All relevant documentation has been updated.

### CODE REVIEW

File: utility/importer/chrome_importer.cc [hunk 1]
```diff
@@ -86,11 +86,14 @@ void ChromeImporter::StartImport(const importer::SourceProfile& source_profile,
     bridge_->NotifyItemEnded(importer::PASSWORDS);
   }
 
+// issues/7044: We do not import cookies from Chrome.
+#if 0
```
Comment (darkdh) [inline utility/importer/chrome_importer.cc]: if we remove the cookies import code for Firefox, there is no point keeping the code for Chrome

Comment (iefremov) [reply utility/importer/chrome_importer.cc]: ok, removed and left a comment

**Comment by iefremov** [comment]
@darkdh thanks, fixed

File: utility/importer/chrome_importer.cc (hunk 1 above)
Comment (iefremov) [reply utility/importer/chrome_importer.cc]: done

File: utility/importer/chrome_importer.cc [hunk 2]
```diff
@@ -86,11 +86,7 @@ void ChromeImporter::StartImport(const importer::SourceProfile& source_profile,
     bridge_->NotifyItemEnded(importer::PASSWORDS);
   }
 
-  if ((items & importer::COOKIES) && !cancelled()) {
-    bridge_->NotifyItemStarted(importer::COOKIES);
-    ImportCookies();
```
Comment (darkdh) [inline utility/importer/chrome_importer.cc]: seems like you forgot to remove https://github.com/brave/brave-core/blob/b0bd76cc949b003697f5ab1b98307cf93ca349bb/utility/importer/chrome_importer.cc#L347

Comment (iefremov) [reply utility/importer/chrome_importer.cc]: It is still in use by `BraveImporter` - asked about it in the ticket.

Comment (darkdh) [reply utility/importer/chrome_importer.cc]: ah, right, `BraveImporter`

**Comment by iefremov** [comment]
@bridiver please take a look

**Comment by iefremov** [comment]
Agreed with @tomlowenthal to do not touch BraveImporter in this PR

**Comment by fmarier** [comment]
Just to confirm, we are talking about dropping all cookies (first and third-party).

This also came up at the Initiatives meeting where Brendan and David were of the opinion that we should just remove that functionality entirely (as opposed to disabling it by default and letting users enable it if they want) to avoid the privacy and compatibility problems we have seen with these.

**Comment by bsclifton** [comment]
@fmarier correct- we want to remove the ability to import cookies completely 👍 (per @BrendanEich, don't even show the checkbox during manual import)

**Comment by bsclifton** [comment]
Will try to review soon! Important to try both manual import and also import via the brave://welcome wizard

**Comment by bsclifton** [comment]
@iefremov we got the go-ahead to remove Muon import, if you wanted to do that also

Either here or in a separate PR (I'd be OK including with this one, as a distinct commit)

**Comment by bsclifton** [comment]
@iefremov only one problem: getting unit test failure for:
```
1 test failed:
    ChromeImporterTest.ImportCookies (../../brave/utility/importer/chrome_importer_unittest.cc:173)
```

I think we can delete that, push latest, and watch CI 😄

**Comment by iefremov** [comment]
@bsclifton I decided to land BraveImporter removal as a follow-up, since there are too many changes involving removing patches, etc.

**Comment by bsclifton** [comment]
All builds + tests passed; just one lint error. Gonna fix that up and merge 😄

**Pull request merged**